qutebrowser/mainwindow/mainwindow.py

```diff
@@ -412,7 +412,7 @@ def _on_config_changed(self, option):
             self._set_decoration(config.val.window.hide_decoration)
 
     def _add_widgets(self):
-        """Add or readd all widgets to the VBox."""
+        """Add or re-add all widgets to the VBox."""
```

Comment:
Just highlighting this to show that my previous suggestion would be in-line with what was done here.

Reply:
This is resolved but somehow Github does not allow me to mark it as such and/or hide this comment. :shrug: